### ISSUE TITLE
Why not with Vue?

### ISSUE THREAD

**Issue by o100ja** [StefanUp1/es6-quiz-app#1]
Hi,

It occurred to me that this project could reasonably quickly be ported to an SPA using Vue or Angular or ReactNative, ...

Food for Thought :wink:

**Comment by StefanUp1** [comment]
Sorry for the late response, I don't know how I did not see that you opened an issue until now.  I uploaded this projects as examples of different technologies that I used. But I think it is a great idea to use Vue for this one. As a matter of fact, I will do it right now.

Cheers 😃

**Comment by StefanUp1** [comment]
@o100ja I finished Vue version. Here it is if you wanna take a look.
 https://github.com/StefanUp1/vue-quizz-app

**Comment by o100ja** [comment]
Adding something like Vuex could help separate the data from the component. 😉

**Comment by StefanUp1** [comment]
Something like this? 😃  https://github.com/StefanUp1/vue-vuex-quiz-app  @o100ja

**Comment by o100ja** [comment]
More on the line on using https://github.com/vuejs/vuefire :wink:, but good work, none the less. :clap: 

I would rename the /data/data.js file to /data/questions.js, though. 
It might make the future addition of files into the /data directory, less vague in regard to naming them.

Compliments on the speed. :muscle:

**Comment by StefanUp1** [comment]
Will do. 😃 

Just to check if I fully understand your suggestion regarding vuefire. You are saying that I should upload my questions on Firebase and then connect it to the app with vuefire? Or did I misunderstood it completely?  😕 

Thanks a lot for your time and interest 😃  @o100ja

**Comment by o100ja** [comment]
Should? No!

But, if you’re interested in taking a path, which I believe may teach you a good deal of the tech you’re obviously interested in, your welcome to try.

BTW that was exactly what I meant, regarding firebase.

Have a great voyage

**Comment by StefanUp1** [comment]
This is a bit tougher then I thought it will be. Vuex and vuefire combined don't behave as I expected. Nevertheless, I will upload it as soon as I figure it out.

Thank you once again for your comments. 😃

**Comment by StefanUp1** [comment]
TL;DR 
I used VuexFire instead of vuefire to connect Firebase with Vuex.  @o100ja 

Didn't manage to connect vufire with Vuex.

 I am not sure if those two are compatible with each other. Member of Vue.js core team said last year that he doesn't know if they are gonna make them be compatible. https://github.com/vuejs/vuefire/issues/64  

Anyway, I used https://github.com/posva/vuexfire to bind Firebase with Vuex.
Here it is: https://github.com/StefanUp1/VuexFire-quiz-app

If you have a solution for this problem please let me know 😃
